### DISCUSSION TITLE
Security Certs on Faction

### DISCUSSION THREAD

**Discussion by glownd** [factionsecurity/faction#57] [General]
Is there a proper built-in or documented way to add security certs onto Faction that I'm overlooking?  Currently proxying by adding a nginx proxy to the docker-compose file, but it's less than ideal for a number of reasons.

**Comment by summitt** [comment]
We do not currently have documentation for implementing this. However, it can be done by modifying the Dockerfile to include your custom certs into the Tomcat configuration.

**Reply by glownd** [reply]
Thanks for the response.  I'll go that route and try to write some documentation for it within the next few weeks.

**Reply by summitt** [reply]
@glownd Please feel free to add it to our documentation portal with a pull request. That can be done by adding a markdown file [here](https://github.com/factionsecurity/factionsecurity.github.io/tree/main/docs)
